index.html

```diff
@@ -282,6 +282,14 @@ <h3>Support</h3>
                   <td>&mdash;</td>
                   <td>&mdash;</td>
                 </tr>
+                <tr>
+                  <th>Regexp Acceptor</th>
+                  <td>&mdash;</td>
+                  <td>&mdash;</td>
+                  <td><a href="https://github.com/shapesecurity/shift-regexp-acceptor-js/tree/es2016">JS</a></td>
+                  <td><a href="https://github.com/shapesecurity/shift-regexp-acceptor-js/tree/es2016">JS</a></td>
```

Comment:
2017?

Reply:
There were no changes to regular expressions in ES2017, hence the branch hasn't changed.